lampshade.go

```diff
@@ -74,74 +74,109 @@
 //   To initialize a session, the client sends the below, encrypted using
 //   RSA OAEP using the server's PK:
 //
-//     +---------+-----+---------+--------+--------+---------+---------+
-//     | Version | Win | Max Pad | Cipher | Secret | Send IV | Recv IV |
-//     +---------+-----+---------+--------+--------+---------+---------+
-//     |    1    |  4  |    1    |    1   | 16/32  |  16/12  |  16/12  |
-//     +---------+-----+---------+--------+--------+---------+---------+
+//     +---------+-----+---------+--------+--------+----------+----------+----------+----------+
+//     | Version | Win | Max Pad | Cipher | Secret | Send IV1 | Send IV2 | Recv IV1 | Recv IV2 |
+//     +---------+-----+---------+--------+--------+----------+----------+----------+----------+
+//     |    1    |  4  |    1    |    1   |   32   |    12    |    12    |    12    |    12    |
+//     +---------+-----+---------+--------+--------+----------+----------+----------+----------+
 //
 //       Version - the version of the protocol (currently 1)
 //
-//       Win     - transmit window size in # of frames
+//       Win        - transmit window size in # of frames
 //
-//       Max Pad - maximum random padding
+//       Max Pad    - maximum random padding
 //
-//       Cipher  - 1 = None, 2 = AES128_CTR, 3 = ChaCha20
+//       Cipher     - specifies the AEAD cipher used for encrypting frames
 //
-//       Secret  - 128 bits of secret for AES128_CTR, 256 bits for ChaCha20
+//                      1 = None
+//                      2 = AES128_GCM
+//                      3 = ChaCha20_poly1305
 //
-//       Send IV - initialization vector for messages from client -> server,
-//                 128 bits for AES_CTR, 96 bits for ChaCha20
+//       Secret     - 256 bits of secret (used for Len and Frames encryption)
 //
-//       Recv IV - initialization vector for messages from server -> client,
-//                 128 bits for AES_CTR, 96 bits for ChaCha20
+//       Send IV1/2 - 96 bits of initialization vector. IV1 is used for
```

Comment:
I was just trying to keep the overall message format to within 80 lines for nice display in godoc.